java/backup/pom.xml

```diff
@@ -32,7 +32,6 @@
             <version>24.1-jre</version>
         </dependency>
 
-        <!--TODO: change to BOM style to only include s3 deps -->
```

Comment:
This TODO still applies though, right? We don't really want to pull in all the other frameworks...

Reply:
Ah, yup it does. I'll put it back in.  

I did give it a go for 5 minutes, but stuff broke and I didn't want to fight maven and dependencies. So will look at soon.